torch/lib/THPP/Traits.hpp

```diff
@@ -83,16 +90,16 @@ template<>
 struct type_traits<unsigned long> {
   static constexpr Type type = Type::ULONG;
   static constexpr bool is_floating_point = false;
-};
+};*/
 
 template<>
-struct type_traits<long long> {
+struct type_traits<int64_t> {
   static constexpr Type type = Type::LONG_LONG;
```

Comment:
I'm actually unsure if it matters to let `int64_t` be of `Type::LONG_LONG` instead of `Type::LONG`? cc @apaszke on this, as I don't know how THPP interacts with the rest of PyTorch